.github/workflows/build.yml

```diff
@@ -1,18 +1,8 @@
+#  PDX-License-Identifier: Apache-2.0
```

Comment:
Missing a leading `S` in all modified headers

Reply:
Sorry for that